karrot/groups/migrations/0027_auto_20180309_0936.py

```diff
@@ -17,6 +17,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='group',
             name='status',
-            field=models.CharField(choices=[('active', 'active'), ('inactive', 'inactive'), ('playground', 'playground')], default='active', max_length=100),
+            field=models.CharField(choices=[('active', 'Active'), ('inactive', 'Inactive'), ('playground', 'Playground')], default='active', max_length=100),
```

Comment:
What is the significance of changing the casing here? (and for the activity type status)

Reply:
First value is the actual one, second one is the "human-readable" one, e.g. for auto-generated dropdown fields in django admin interface. I believe this change shouldn't affect us, it's just a side effect from switching from `Enum`s to Django's `TextChoices`.